model/gen-ai/registry.yaml

```diff
@@ -90,6 +90,18 @@ groups:
         type: int
         brief: The maximum number of tokens the model generates for a request.
         examples: [100]
+      - id: gen_ai.request.min_response_candidates
```

Comment:
are there any APIs that have range? if not, I'd avoid adding this attribute:

https://github.com/open-telemetry/semantic-conventions/blob/c1f8e0a86410560da33f8cf3f18237ddf819fa1a/docs/general/how-to-define-semantic-conventions.md?plain=1#L50C8-L50C9

We can always add one later if/when the need comes

Reply:
Fair point. Switched to a single target value.